environments/staging-phase-two/files/versions.cfg

```diff
@@ -0,0 +1,99 @@
+[buildout]
+versions = versions
+newest = true
+
+[versions]
+Paste = 1.7.5.1
+PasteDeploy = 1.5.0
+PasteScript = 1.7.5
+Pillow = 1.7.6
+Products.AdvancedQuery = 2.2-rhaptosdev-r30378
+Products.CMFDiffTool = 0.3.3dev-r30461
+Products.CMFSquidTool = 1.5.1
+Products.CNXCaching = 0.2
+Products.CNXContent = 0.47
+Products.CNXMLDocument = 0.49
+Products.CNXMLTransforms = 0.23
+Products.CNXPloneSite = 0.76.2dev
+Products.CacheSetup = 1.2.1
+Products.CatalogMemberDataTool = 1.5
+Products.ClockServer = 0.2-Zope2.9dev
+Products.ExtZSQL = 0.9
+Products.ExternalFile = 0.1-rhaptosdev-r30470
+Products.ExternalStorage = 0.8.1
+Products.FSImportTool = 0.8
+Products.FeatureArticle = 0.10.1
+Products.Five = 1.4.5-rhaptosdev-r32037
+Products.LensOrganizer = 0.3.1
+Products.Lensmaker = 0.24.1dev
+Products.Lineup = 0.6
+Products.LinkMapTool = 0.9
+Products.LocalFS = 1.7-rhaptosdev-r30378
+Products.ManagableIndex = 1.7.3-rhaptosdev-r30378
+Products.MasterSelectWidget = 0.2.3-rhaptosdev-r30378
+Products.MathEditor = 0.4.1
+Products.NoHeaderFieldContinuation = 0.1-rhaptosdev-r30378
+Products.OFolder = 1.0-rhaptosdev-r30378
+Products.PageCacheManager = 1.2
+Products.PloneHotfix20110531 = 2.0
+Products.PloneHotfix20110720 = 1.2
+Products.PloneHotfix20121106 = 1.2
+Products.PolicyHTTPCacheManager = 1.2
+Products.References = 0.10-rhaptosdev-r30378
+Products.RhaptosBugTrackingTool = 0.1.3
+Products.RhaptosCacheTool = 0.2.1
+Products.RhaptosCatFixes = 0.2
+Products.RhaptosCollaborationTool = 0.10.1
+Products.RhaptosCollection = 0.75
+Products.RhaptosContent = 0.66
+Products.RhaptosHitCountTool = 0.12
+Products.RhaptosModuleEditor = 0.83
+Products.RhaptosModuleStorage = 1.1.2
+Products.RhaptosPDFLatexTool = 0.15.1dev
+Products.RhaptosPatchTool = 0.7.2
+Products.RhaptosPrint = 0.90
```

Comment:
There may have been a release of RhaptosPrint since I last updated this file. This is likely one version pinning to verify.